e2e-tests/webdriver-helpers.ts

```diff
@@ -0,0 +1,365 @@
+/**
+ * Copyright 2019 The Ground Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Browser,
+  Builder,
+  By,
+  Key,
+  WebDriver,
+  WebElement,
+  until,
+} from 'selenium-webdriver';
+import {TestConfig} from './test_config.js';
+import {LoiType, Role, TaskType} from './ground-helpers.js';
+
+class WebDriverHelperException extends Error {}
+
+function driverInitialized(
```

Comment:
Nit: Can we update the name to make it clear this fails fast when something isn't true? ie `assertWebDriverInitialized()`

Reply:
Done.